quickstart/201-iot-hub-with-device-provisioning-service/providers.tf

```diff
@@ -0,0 +1,18 @@
+terraform {
+  required_version = ">=0.12"
+
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~>2.0"
```

Comment:
The mainstream version of `azurerm` provider is 3.x now and 2.x is no longer maintained anymore, would the following restriction be better?:

```hcl
version = ">=3.0"
```